workflow/rules/bwa_mem.smk

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule bwa_mem:
+    input:
+        reads=["pre-alignment/merge/{sample}_{unit}_R1.fq.gz", "pre-alignment/merge/{sample}_{unit}_R2.fq.gz"],
```

Comment:
```suggestion
        reads=["prealignment/merged/{sample}_{unit}_fastq1.fq.gz", "prealignment/merged/{sample}_{unit}_fastq2.fq.gz"],
```

Path that prealignment module uses and also changed fq to fastq since we have that in the unit file.